src/js/api/users/editStudent.js

```diff
@@ -1,28 +1,38 @@
 import { apiPath } from '../constants.js';
+import { getToken } from '../getToken.js';
 
 const method = 'PUT';
 const action = 'users/';
 
 /**
- * This function sends a PUT request to the API to update the company profile 
- * 
+ * This function sends a PUT request to the API to update the company profile
+ *
  * @param {Object} profile The updated user profile data
  * @returns {Promise<Object>} A Promise that resolves with the updated user profile
- * @throws {Error} If the 'id' is missing, the API request fails, or if it returns an error status 
+ * @throws {Error} If the 'id' is missing, the API request fails, or if it returns an error status
  */
 
 export async function editStudent(profile) {
-  const { id } = profile;
-  const profileURL = apiPath + action + `${id}`;
+  // const { id } = profile;
+  const id = localStorage.getItem('id');
+  const newId = id.replace(/^"|"$/g, '');
+
+  const profileURL = apiPath + action + newId;
+
+  console.log('hi', profile);
+  const accessToken = getToken('token');
+  const newAccessToken = accessToken.replace(/^"|"$/g, '');
+  console.log('accessToken', newAccessToken);
```

Comment:
Especially when it comes to accesstokens

Reply:
Sorry, forgot about it, will keep an eye out for it going forward.